src/contexts/Users.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const UserContext = createContext();
```

Comment:
This context is incomplete... have another look at this before submitting it for review